### PR TITLE
Taskcluster indexes

### PR DESCRIPTION
**Pull request by mitchhentges**
Also adds staging builds, which were blocking me from testing the new indexes

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -216,15 +223,25 @@ tasks:
...
-          - queue:create-task:highest:scriptworker-prov-v1/mobile-signing-v1
-          - queue:create-task:highest:scriptworker-prov-v1/mobile-pushapk-v1
-          - project:mobile:focus:releng:signing:cert:release-signing
-          - project:mobile:focus:releng:signing:format:autograph_focus
-          - project:mobile:focus:releng:googleplay:product:focus
-          - secrets:get:project/focus/tokens
-          - queue:route:index.project.mobile.focus.nightly.latest
+          $flatten:
+            - queue:scheduler-id:${scheduler_id}
+            - queue:create-task:highest:aws-provisioner-v1/gecko-focus
```
Comment (MihaiTabara) [inline .taskcluster.yml]: I think this is defunct now. Or it's about to be. Might be a good moment to update this too.

File: tools/taskcluster/release.py [hunk 2]
```diff
@@ -38,66 +41,63 @@ def generate_build_task(apks, tag):
...
 
-
     return taskcluster.slugId(), BUILDER.build_task(
         name="(Focus for Android) Build task",
         description="Build Focus/Klar from source code.",
         command=(checkout +
-                 ' && python tools/taskcluster/get-adjust-token.py'
-                 ' && python tools/taskcluster/get-sentry-token.py'
+                 ' && python tools/taskcluster/get-adjust-token.py --staging'
+                 ' && python tools/taskcluster/get-sentry-token.py --staging'
```
Comment (JohanLorenzo) [inline tools/taskcluster/release.py]: Nit: Use `is_staging` to determine if `--staging` should be appended.

File: tools/taskcluster/schedule-nightly-graph.py [hunk 3]
```diff
@@ -68,17 +78,26 @@ def as_slugid(name):
...
-if __name__ == "__main__":
+def schedule(is_staging):
     queue = taskcluster.Queue({ 'baseUrl': 'http://taskcluster/queue/v1' })
 
-    branch, head_rev = calculate_branch_and_head_rev(ROOT)
-
+    html_url, branch, head_rev = calculate_git_references(ROOT)
     params = {
-        'repository_url': 'https://github.com/mozilla-mobile/focus-android',
+        'is_staging': is_staging,
```
Comment (JohanLorenzo) [inline tools/taskcluster/schedule-nightly-graph.py]: Nit: I don't see this parameter used in `make_decision_task()`. Let's remove it.

File: .taskcluster.yml [hunk 4]
```diff
@@ -245,28 +262,30 @@ tasks:
...
-              && git reset --hard origin/master
+              cd ..
+              && git clone ${repository} repository
+              && cd repository
               && python tools/taskcluster/release.py \
-                --track nightly \
+                --track ${track} \
                 --commit \
-                --output /opt/focus-android/app/build/outputs/apk \
+                --output /opt/repository/app/build/outputs/apk \
```
Comment (JohanLorenzo) [inline .taskcluster.yml]: I'm surprised changing this folder doesn't require a new docker image, but I confess I didn't look much into it :)

Comment (mitchhentges) [reply .taskcluster.yml]: I also changed the line above: https://github.com/mozilla-mobile/focus-android/pull/4129/files/c081985cdf6c936d2083afce81ac2afd61ed52b6#diff-ac0229d1171c0983b27003af4c7441a5R266

**Pull request merged**